### PR TITLE
[READY] Current domain handles subdomain for multi tenant set true

### PR DESCRIPTION
**Pull request by michalsz**
- [x] current_subdomain returns correct value if Hours.single_tenant_mode? is false
- [x] user can create account if Hours.single_tenant_mode? is true
- [x] current_subdomain returns correct value if Hours.single_tenant_mode? is true

### CODE REVIEW

File: app/helpers/email_helper.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module EmailHelper
   def current_subdomain
-    Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current 
+    if Hours.single_tenant_mode?
+      ""
+    else
+      Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current
+    end
```
Comment (jurre) [inline app/helpers/email_helper.rb]: What about using a guard clause?

```ruby
def current_subdomain
  return "" if Hours.single_tenant_mode?
  Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current
end
```

Comment (michalsz) [inline app/helpers/email_helper.rb]: In single tenant mode we need to check if application uses domain if yes then we need to return the subdomain, if not the we return ''. So is possible that it won't be one line condition syntax .

Comment (fatboypunk) [inline app/helpers/email_helper.rb]: The code that Jurre supplied does the same as your code in that diff. So in two lines it will work :)

Comment (michalsz) [inline app/helpers/email_helper.rb]: Yes I need check if application is set SIGNLE_TENANT_MODE to true and it uses subdomain ie
hourstracker.superglobalcompany.com

File: app/models/account.rb [hunk 2]
```diff
@@ -37,4 +39,8 @@ class Account < ActiveRecord::Base
   def downcase_subdomain
     self.subdomain = subdomain.try(:downcase)
   end
+
+  def single_tenant_mode?
+    Hours.single_tenant_mode?
+  end
```
Comment (jurre) [inline app/models/account.rb]: Can't we just call `Hours.single_tenant_mode?` wherever we need to know this?

Comment (michalsz) [inline app/models/account.rb]: The single_tenant_mode? from Account is used in validation for subdomain - we don't need subdomain for account if app works in single tenatnt mode. Sadly Hours.single_tenant_mode? dosen't work for the validation. Do you prefer move the validation to custom validator and in the new validator use Hours.single_tenant_mode? method ?

Comment (jurre) [inline app/models/account.rb]: Right, missed that you're using this in the validation here, this is fine then! :)

**Comment by jurre** [comment]
We've decided to freeze feature development as none of the maintainers currently have time to properly steward contributions from the community.

**Pull request closed without merging**